src/cudamatrix/cu-matrix.h

```diff
@@ -299,6 +303,11 @@ class CuMatrixBase {
   /// *this = tanh(src).
   void Tanh(const CuMatrixBase<Real> &src);
 
+  /// Differentiate backward through the relu function.  Here, "value" is the
```

Comment:
Hi, it would be more logical if 'value' was the input of original ReLU function. Like this the could would work even with the negative 'a's or 'b's.